libs/dh/dev-examples/feature-processes/graphql/get-processes.graphql

```diff
@@ -23,6 +23,28 @@ query GetProcesses(
         state
         isCurrent
       }
+      ... on RequestCalculatedWholesaleServicesResult {
+        __typename
+        priceType
+        period
+        calculationType
+      }
+      ... on RequestCalculatedEnergyTimeSeriesResult {
+        __typename
+        meteringPointType
+        calculationType
+        period
+      }
+      ... on Calculation {
+        __typename
+        gridAreas {
+          id
+          displayName
+        }
```

Comment:
suggestion: for the sake of the example, you could consider not fetching gridAreas for the table view (since it is an additional request) and instead only fetch them when opening the details view

Reply:
Fixed in https://github.com/Energinet-DataHub/greenforce-frontend/pull/3958/commits/131eb0512bd099ad65160cd7c597c7c77edb6a4b